src/IntegratedExperts/BehatScreenshotExtension/Context/ScreenshotContext.php

```diff
@@ -117,16 +119,29 @@ public function iSaveScreenshot()
         $driver = $this->getSession()->getDriver();
 
         if ($driver instanceof Selenium2Driver) {
-            $data = $this->getSession()->getScreenshot();
-            $ext = 'png';
+            // If both png and html are not set assume the default of saving png screen shots.
```

Comment:
This comment is not quite truthful in this context since both of the variables were set in `setScreenshotParameters` to the values passed from the parameters.

There is also no need to to check for empty as `$this->png` and `$this->html` will always be either `true` or `false` and will be always populated (see above).